Makefile

```diff
@@ -12,6 +12,7 @@ deps:
 	go get -u github.com/elazarl/go-bindata-assetfs/...
 	go get -u github.com/dchest/jsmin
 	go get -u github.com/franela/goblin
+	go get -u github.com/mrjones/oauth
```

Comment:
also can you remove this line?

we vendor our dependencies using the `govendor` tool. You can download the govendor tool with the following command:

```
go get -u github.com/kardianos/govendor
```

and then you can add the oauth dependency with the following line: 

```
govendor add github.com/mrjones/oauth
```

note that if this is vendored properly you should only see mrjones/oauth added to the vendor directory and nothing else